subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/builder/EdgeState.java

```diff
@@ -238,6 +238,12 @@ public Long getSelected() {
         return getSelectedComponent().getResultId();
     }
 
+    @Override
+    public boolean isSelectedAVirtualPlatform() {
```

Comment:
:o: strange method name :p Maybe `isSelectedAndVirtualPlatform()` ?

Reply:
Not exactly what I meant. What about `isTargetAVirtualPlatform()`?

Reply:
It's the "A" that puzzles me :)

Reply:
Ok, will settle for `isTargetVirtualPlatform()` then 😉